--- hsftp-1.15/makefile.in~	2004-02-16 14:38:17.000000000 +0100
+++ hsftp-1.15/makefile.in	2012-10-27 06:41:14.680114995 +0200
@@ -36,7 +36,7 @@ AUTOMAKE = @AUTOMAKE@
 AUTOHEADER = @AUTOHEADER@
 
 INSTALL = @INSTALL@
-INSTALL_PROGRAM = @INSTALL@ -s -m 755
+INSTALL_PROGRAM = @INSTALL@ -m 755
 INSTALL_DATA = @INSTALL@ -m 444
 
 transform = @program_transform_name@
@@ -196,7 +196,7 @@ install-nobinDATA:
 	fi
 
 install-strip:
-	$(MAKE) INSTALL_PROGRAM='$(INSTALL_PROGRAM) -s' INSTALL_SCRIPT='$(INSTALL_PROGRAM)' install
+	$(MAKE) INSTALL_PROGRAM='$(INSTALL_PROGRAM)' INSTALL_SCRIPT='$(INSTALL_PROGRAM)' install
 
 installdirs: installdirs-recursive
 
